bigtop-packages/src/rpm/spark/SPECS/spark.spec

```diff
@@ -116,11 +116,7 @@ Server for Spark worker
 %package -n %{spark_pkg_name}-python
 Summary: Python client for Spark
 Group: Development/Libraries
-%if 0%{?rhel} >= 8
-Requires: %{spark_pkg_name}-core = %{version}-%{release}, python2
-%else
-Requires: %{spark_pkg_name}-core = %{version}-%{release}, python
-%endif
+Requires: %{spark_pkg_name}-core = %{version}-%{release}, python36
```

Comment:
Currently, bigtop uses Spark 3.2, but since Spark 3.3, the minimal python version is 3.7, so can we use a higher version to make it more future-proofing?
https://spark.apache.org/docs/3.3.2/api/python/getting_started/install.html

Reply:
yes, we can.